embeddings/get_started_with_embedding_tuning.ipynb

```diff
@@ -0,0 +1,1434 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #99.        output_artifact = MessageToDict(trainer_task.execution._pb)["metadata"]
</pre><p>Does this variable get overwritten?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
done